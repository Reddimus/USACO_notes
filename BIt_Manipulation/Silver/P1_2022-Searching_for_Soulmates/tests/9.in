10
1044394699 4199286377
3701731062 1603249190
3696154266 4072183716
3336908514 2151840192
2138246396 3459472733
844525223 2501267026
3197975454 1033979701
1617707432 557790053
1324549298 921993608
2163687123 444710612
