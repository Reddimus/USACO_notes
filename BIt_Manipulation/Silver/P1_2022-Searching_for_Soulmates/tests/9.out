67
84
82
80
87
82
79
86
80
81
